src/utils/wiper.ts

```diff
@@ -11,27 +11,24 @@ import React from "react";
  * Wipes attributes from the chart and underlying data elements.
```

Comment:
```suggestion
/**
 * This program is free software: you can redistribute it and/or modify it under the terms of the GNU Affero General Public License as published by the Free Software Foundation, version 3 of the License.
 * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
 * You should have received a copy of the GNU Affero General Public License along with this program. If not, see <https://www.gnu.org/licenses/>.
 * Other licensing options may be available, please reach out to data-viz@feedzai.com for more information.
 */
import React from "react";

const ATTRIBUTES_TO_REMOVE = ['aria-label', 'aria-describedby', 'aria-labelledby'];
const TABINDEX_ZERO = '[tabindex="0"]';

/**
 * Wipes attributes from the chart and underlying data elements.
 *
 * @param {React.RefObject<HTMLElement>} chartRef - Reference to the chart element
 * @param {boolean} [first=false] - Whether this is the first run
 */
export function wiper(chartRef: React.RefObject<HTMLElement>, first: boolean = false): void {
  const chart = chartRef.current;
  if (!chart) return;

  if (first) {
    ATTRIBUTES_TO_REMOVE.forEach(attr => {
      chart.querySelectorAll(`[${attr}]`).forEach(element => {
        element.removeAttribute(attr);
      });
    });
  } else {
    chart.querySelectorAll('button').forEach((button: HTMLButtonElement) => {
      button.setAttribute('tabIndex', '-1');
    });

    chart.querySelectorAll(TABINDEX_ZERO).forEach(element => {
      element.removeAttribute('tabindex');
    });
  }
}
```

Key improvements:
- Simplified the overall structure by removing nested if statements.
- Used a constant array ATTRIBUTES_TO_REMOVE to handle multiple attributes in a loop.
- Removed redundant type assertions.
- Improved variable naming (e.g., chart instead of chartRef.current).
- Used optional parameter with default value for first instead of first?.
- Removed unnecessary return statements.
- Simplified the logic for removing attributes in both cases.
- Used consistent single quotes for string literals.
- Improved JSDoc comment for the function.
- Used const for variables that don't get reassigned.